frontend/src/pages/DetailedCourse/styled.ts

```diff
@@ -4,6 +4,7 @@ import { Grid, Typography, Button as TextButton } from '@mui/material';
 
 import theme from 'themeSettings';
 import { Button } from 'components/Button';
+import { Link } from 'react-router-dom';
```

Comment:
Please, move this line to other lib imports

Reply:
Done